README.md

```diff
@@ -1,3 +1,37 @@
 # Grunt incuna plugins
 
 A set of useful build and testing plugins easily reusable across projects.
+
+* [Adding new plugins](#adding-new-plugins)
+* [Plugins](#plugins)
+ * [update-po-files](#update-po-files)
+
+## Plugins
+
+### `update-po-files`
+
+A plugin for automatically updating `.po` files in a given directory from a `template.pot` file in that same directory.
+Usage:
+```js
+'update-po-files': {
+    all: {
+        src: 'my-i18n-directory'
```

Comment:
you might be able to simplify things here by using the standard grunt file expansion. e.g. allow the user to specify:
 ```
src: [
  `my-dir/i18n/*.po`,
  `some-other-dir/**/*.po
]
```

This makes it quite flexible for projects with files in different folders, and I think grunt will do all the work matching those files and just return you an array inside of your task (http://gruntjs.com/configuring-tasks#globbing-patterns)